Source/WebCore/css/query/MediaQueryFeatures.cpp

```diff
@@ -689,6 +689,40 @@ const FeatureSchema& displayMode()
 }
 #endif
 
+const FeatureSchema& overflowBlock()
+{
+    static MainThreadNeverDestroyed<IdentifierSchema> schema {
+        "overflow-block"_s,
+        Vector { CSSValueNone, CSSValueScroll, CSSValuePaged },
```

Comment:
`Vector` is not the ideal data structure for something we do not intend to resize. Maybe `std::initializer_list<CSSValueID>` is what `FeatureSchema` should use, or maybe `FixedVector`.

related, do we actually have to write `Vector` here or can we just omit the class template name/ If we can omit it, that makes future refactoring easier.

Reply:
I can try omitting `Vector` and refactoring `FeatureSchema` away from `Vector` in a separate patch.

Reply:
If I omit `Vector` the compiler can't automatically infer the type of the schema